cam0:
  # Hamilton
  T_cam_imu:
    [-0.0029580312299853917, 0.9999838756153343, 0.004847531390036592, 0.043127511338890666,
    -0.999983882585975, -0.0029344699459535883, -0.004860396528509914, -0.002703158935029758,
    -0.004846093222430533, -0.004861830465087627, 0.9999764387149367, -0.04566042599888088,
    0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.6383007675961103, 0.3839940950610235, -0.0026243087644044843,0.0005602619236159766]
  distortion_model: radtan
  intrinsics: [1309.6717671914262, 1311.1638366466414, 382.0527762678694, 266.55550108289566]
  resolution: [752, 480]
  timeshift_cam_imu: 0.007029219996500052
cam1:
  # Hamilton
  T_cam_imu:
    [-0.0047296462980803144, 0.9999710752569572, 0.005956433105469543, -0.07682092270861207,
    -0.9999819370242012, -0.0047074374477888425, -0.0037370654264682213, -0.002536558352956211,
    -0.00370891779655509, -0.005974000512222492, 0.9999752773177237, -0.04557754623915776,
    0.0, 0.0, 0.0, 1.0]
  # Hamilton
  T_cn_cnm1:
    [0.9999978157765939, 0.0017662343744603072, 0.0011175232332301006, -0.1198925388482594,
    -0.0017674992860792203, 0.9999977973548442, 0.0011319150739576846, 0.00029450619794693704,
    -0.00111552154441037, -0.0011338878231191968, 0.9999987349542445, 0.0001278665862499424,
    0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.6323876403580783, 0.33424998370866404, -0.0019091995355724667, -0.00033212257126655556]
  distortion_model: radtan
  intrinsics: [1308.5056941536666, 1310.2788132101373, 375.1306284695755, 259.05573800729564]
  resolution: [752, 480]
  timeshift_cam_imu: 0.006894213857692347
# is_svd Hamilton
T_gps_imu:
  [-0.0095936300395616,  0.0037362959735642,  0.9999469997730189, 1.500000000000000,
    0.0070449460125148,  0.9999684538750927, -0.0036687859487000, 0.0000000000000000,
  -0.9999291629902315,  0.0070093756536908, -0.0096196494011305, -0.5500000000000000,
  0.0000000000000000, 0.0000000000000000, 0.0000000000000000,   1.0000000000000000]
# bs_manual Hamilton
T_gps_body:
  [1.0000, 0.0000, 0.0000, 0.0000,
  0.0000, -1.0000, 0.0000, 0.0000,
  0.0000, 0.0000, -1.0000, 0.0000,
  0.0000, 0.0000, 0.0000, 1.0000]

ackermann_kv: 0.0
ackermann_ks: 0.0
ackermann_bs: 0.0
ackermann_bl: 0.0
ackermann_bt: 0.0
ackermann_WheelBase: 1.9
ackermann_TireBase: 0.98
